tests/endtoend/test_blob_functions.py

```diff
@@ -132,11 +128,10 @@ def test_blob_trigger(self):
 
     @testutils.retryable_test(3, 5)
     def test_blob_trigger_with_large_content(self):
-        datetime_iso = datetime.utcnow().isoformat()
-        data = datetime_iso * 1024 * 1024  # 26 MB
+        data = 'DummyDataDummyDataDummyData' * 1024 * 1024  # 27 MB
```

Comment:
Python 3.9 becomes very slow and sometimes, previous runs have their files remaining in the system. I'll be moving the date-time logic to a guid-based logic. This is an intermediate step for now to remove reliance on date-times in the tests.

Reply:
Hmm, this is interesting, where does this slow comes from? It is when running this test_blob_io_large_bytes testcase? Or does it comes from datetime.utcnow()? I don't mind the intermediate step for changing it to static string for now, but this still introduce another task to move it back to guid eventually.